hypha/apply/projects/models/project.py

```diff
@@ -290,6 +296,15 @@ def create_from_submission(cls, submission, lead=None):
             )
             return None
 
+        # The PROJECTS_INITIAL_STATUS env var allows a funder to skip straight to a given Project stage aka status.
+        # Note this is case-sensitive and all the choice keys are lower case. So use a lower-case string setting.
+        status = DEFAULT_PROJECT_STATUS
+        if settings.PROJECTS_INITIAL_STATUS and any(
+            CHOICE[0] == str(settings.PROJECTS_INITIAL_STATUS)
+            for CHOICE in PROJECT_STATUS_CHOICES
+        ):
+            status = settings.PROJECTS_INITIAL_STATUS
```

Comment:
@bickelj Can you please remove all code regarding `PROJECTS_INITIAL_STATUS`. We will instead add settings in a separate PR that allow you to disable state by state as needed. More flexible in the long run is our reasoning.

Reply:
@frjo Thank you for this suggestion. Settings is the better choice for the long run. In the process of removal I found some missing test-related commits that are now included.